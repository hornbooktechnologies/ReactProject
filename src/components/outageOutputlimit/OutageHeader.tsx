import React from 'react'
import { OutageHeaderModel } from '../../models/outageOutputLimit/OutageHeaderModel'
const OutageHeader = ({ title }: OutageHeaderModel) => {
  return <h5 className="mb-24 font-18">{title}</h5>
}

export default OutageHeader
