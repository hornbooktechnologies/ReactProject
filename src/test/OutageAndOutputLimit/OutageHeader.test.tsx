import React from "react";
import renderer from "react-test-renderer";
import OutageHeader from "../../components/outageOutputlimit/OutageHeader";

describe("Component: OutageHeader", () => {
  it("OutageHeader renders correctly", () => {
    const tree = renderer.create(<OutageHeader title="OutageHeader"/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
