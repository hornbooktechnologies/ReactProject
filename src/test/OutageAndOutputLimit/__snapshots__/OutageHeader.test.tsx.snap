// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: OutageHeader OutageHeader renders correctly 1`] = `
<h5
  className="mb-24 font-18"
>
  OutageHeader
</h5>
`;
